modules/effects/arrow-effects-data/src/main/kotlin/arrow/effects/typeclasses/ConcurrentCancellableContinuations.kt

```diff
@@ -0,0 +1,23 @@
+package arrow.effects.typeclasses
+
+import arrow.Kind
+import arrow.effects.typeclasses.suspended.ConcurrentSyntax
+import arrow.typeclasses.MonadContinuation
+import kotlin.coroutines.CoroutineContext
+import kotlin.coroutines.EmptyCoroutineContext
+import kotlin.coroutines.RestrictsSuspension
+
+//typealias Fx<F> = ConcurrentCancellableContinuation<F, *>
```

Comment:
Ded code. Please delet.

Reply:
Addressed in https://github.com/arrow-kt/arrow/pull/1263/commits/6ab8081ad7b6bb3d605f7eed806ab8660e8ef8ad